src/Repositories/Behaviors/HandleBrowsers.php

```diff
@@ -28,6 +29,18 @@ trait HandleBrowsers
      */
     protected $browsers = [];
 
+    /**
+     *  Adding browsers names activates the create option on the browser
+     *
+     * [
+     *   'books',
+     *   'publications'
+     * ]
+     *
+     *
+     */
+    protected $browsersWithCreate = [];
```

Comment:
All browsers will have the create option activated by default, unless some browsers are specified in `$browsersWithCreate`, am I understanding this correctly?

I'm curious to know, have you explored other ways to pass this configuration, like on the browser form field itself, or as an option of the "array" mode of `$browsers`?